tests/test_infra.py

```diff
@@ -29,20 +29,29 @@
 FLAKE8_OPTIONS = {'max_line_length': 99, 'select': ['E', 'W', 'F', 'C', 'N']}
 
 
-def _get_python_filepaths():
+def get_python_filepaths(*, roots=None, python_paths=None):
     """Helper to retrieve paths of Python files."""
-    python_paths = ['setup.py']
-    for root in ['charmcraft', 'tests']:
+    if python_paths is None:
+        python_paths = ['setup.py']
+    if roots is None:
+        roots = ['charmcraft', 'tests']
+    for root in roots:
         for dirpath, dirnames, filenames in os.walk(root):
             for filename in filenames:
                 if filename.endswith(".py"):
                     python_paths.append(os.path.join(dirpath, filename))
+            for i, dirname in enumerate(dirnames):
```

Comment:
revisit all this after the `.j2` renaming

Reply:
we still need all these changes?

Reply:
probably not

Reply:
removed it